### PR TITLE
Integration Cache - update with wrapper funcs

### PR DESCRIPTION
**Pull request by Itay4**
## Status
Ready

## Description
update the integration cache doc with the relevant wrapper funcs to use

### CODE REVIEW

File: docs/integrations/integration-cache.md [hunk 1]
```diff
@@ -71,9 +71,9 @@ integration_context = {
...
 }
-demisto.setIntegrationContext(integration_context)
+set_integration_context(integration_context)
 ```
 
 ### For more examples, refer to following integrations:
  - [Microsoft Graph](https://github.com/demisto/content/blob/master/Packs/ApiModules/Scripts/MicrosoftApiModule/MicrosoftApiModule.py)
- - [EWS v2](https://github.com/demisto/content/blob/master/Packs/EWS/Integrations/EWSv2/EWSv2.py) 
\ No newline at end of file
+ - [EWS v2](https://github.com/demisto/content/blob/master/Packs/EWS/Integrations/EWSv2/EWSv2.py)
```
Comment (ShahafBenYakir) [inline docs/integrations/integration-cache.md:79]: Both EWS v2 and Microsofts Graph use the ole function, need to change this as well

**Review by ShahafBenYakir — APPROVED** [review]

File: docs/integrations/integration-cache.md (hunk 1 above)
Comment (Itay4) [reply docs/integrations/integration-cache.md:79]: graph uses the wrappers, so its good
ill replace ews with snow

File: docs/integrations/integration-cache.md [hunk 2]
```diff
@@ -40,24 +40,24 @@ Note that this method overrides the existing object which is stored, so in order
 
 ### General usage
 ```python
-integration_context: Dict = demisto.getIntegrationContext()
+integration_context: Dict = demisto.get_integration_context()
```
Comment (anara123) [inline docs/integrations/integration-cache.md:43]: isn't it `integration_context: Dict = get_integration_context()` without the `demisto`

Comment (Itay4) [reply docs/integrations/integration-cache.md:43]: good catch, will open a pr to fix this

**Pull request merged**